### ISSUE TITLE
[Player/Spells] Hunter Starting Pets not spawning

### ISSUE THREAD

**Issue by drikishuk** [mfranca0009/Origins-Issue-Tracker#24]
Hunter starter pets are no longer spawning for some reason, this was working before. 
Could be creature_template? They are summoned by spells.

**Comment by drikishuk** [comment]
@mfranca0009 - do you think this could be anything to do with your core work recently? They were working before but could also be my stuff.

**Comment by mfranca0009** [comment]
No, hunter pets would not have been effected by current changes.
